### PR TITLE
release 2.4.6

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by johanandren** [comment]
ping team @patriknw @drewhk @ktoso

**Comment by ktoso** [comment]
LGTM, ok wording

File: news/_posts/2016-05-19-akka-2.4.6-released.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+soon after [releasing Akka 2.4.5](http://akka.io/news/2016/05/17/akka-2.4.5-released.html) with the new Routing DSL
+for Java it was discovered (thanks [relgames](https://github.com/relgames)) that the `akka-http-experimental` artifact
+was incomplete ([#20556](https://github.com/akka/akka/issues/20556)), and the missing class files did in fact make
+parts of the high-level server APIs unusable.
+
+We are therefore with some urgency releasing a new minor update, Akka 2.4.6, to correct this.
+
+The small list of changes since 2.4.5 is available under the
+[2.4.6](https://github.com/akka/akka/issues?q=milestone%3A2.4.5) milestone on github for your reference.
```
Comment (ktoso) [inline news/_posts/2016-05-19-akka-2.4.6-released.md]: whoops

Comment (ktoso) [inline news/_posts/2016-05-19-akka-2.4.6-released.md]: Also, link to closed tab

Comment (johanandren) [inline news/_posts/2016-05-19-akka-2.4.6-released.md]: fixed directly in master because I have no shame

Comment (ktoso) [inline news/_posts/2016-05-19-akka-2.4.6-released.md]: ;)

**Pull request merged**